pymatbridge/publish.py

```diff
@@ -1,5 +1,5 @@
-import IPython.nbformat.v4 as nbformat
-from IPython.nbformat import write as nbwrite
+import nbformat.v4 as nbformat
```

Comment:
This should be a try/except ImportError with the new format first to accommodate both versions.